src/alfasim_sdk/_internal/cli.py

```diff
@@ -104,6 +109,11 @@ def new(dst, caption, plugin_id, author_name, author_email):
         "}",
     ]
 
+    requirements = {
+        "sdk_version": get_required_sdk_version(),
+        "python_version": get_required_python_version(),
```

Comment:
Have we already committed to these keys? I believe "version" is implied and think we should use `"python"` and `"alfasim_sdk"`.